frontend/app/controllers/Info.scala

```diff
@@ -34,6 +34,10 @@ class Info(
     Redirect(routes.Info.supporterFor(determinedCountryGroup).url, campaignCodes(request), SEE_OTHER)
   }
 
+  def patronsRedirect = NoCacheAction { implicit request =>
+    Redirect(Links.patrons, campaignCodes(request), TEMPORARY_REDIRECT)
```

Comment:
I think MOVED_PERMANENTLY might be more appropriate?

Reply:
I did think about that, but for now (I think) that destination is a proof-of-concept site, and the final resting place may be different.